libsolidity/interface/Natspec.cpp

```diff
@@ -73,33 +81,23 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 	if (!dev.empty())
 		doc["details"] = Json::Value(dev);
 
+	auto constructorDefinition(_contractDef.constructor());
+	if (constructorDefinition) {
+		Json::Value constructor;
+		fillDevDocumentation(constructor, constructorDefinition->annotation().docTags);
```

Comment:
How about `constructor = std::move(fillDevDocument(...)` and `fillDevDocumentation` just return a `Json::Value`? That is a bit more consistent with the style we have.

Reply:
Ok, I change the method to return a Json::Value. But I think that `std::move` is not needed here, because the compiler will already do return value optimization.